superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/TableRenderers.jsx

```diff
@@ -423,7 +423,11 @@ export class TableRenderer extends React.Component {
           dateFormatters &&
           dateFormatters[attrName] &&
           typeof dateFormatters[attrName] === 'function'
-            ? dateFormatters[attrName](colKey[attrIdx])
+            ? dateFormatters[attrName](
```

Comment:
@wangkun723 it doesn't look safe to custom logic in each chart. The logic should be within dateFormatter. 
I also found [this PR](https://github.com/apache/superset/pull/23339) from @villebro which seems cover this case too.
Could you try with the above hotfix?

Reply:
Okay, thank you for your guidance